### DISCUSSION TITLE
Nuclei hanged on Windows 10 x64 after message [INF] Skipped itexpert.work:80 from target list as found unresponsive 30 times

### DISCUSSION THREAD

**Discussion by JuraLys** [projectdiscovery/nuclei#5386] [Q&A] [answered]
start scan itexpert.work target and wait message [INF] Skipped itexpert.work:80 from target list as found unresponsive 30 times
=> Nuclei hanged on Windows 10 x64, see 
![nuclei-scan-hang( jpg](https://github.com/projectdiscovery/nuclei/assets/146434286/ff350c84-d27e-4c77-95b2-36be1f59d2cc)

**Comment by JuraLys** [comment]
nuclei v3.2.8

**Comment by tarunKoyalwar** [comment, marked as answer]
@JuraLys , this was fixed in recent version i believe v3.2.9 maybe , and only happens when http requests are failing the two approaches to fix this is
- increase timeout using `-timeout` to 30 maybe ?
- reduce values of `-c` and `-bs` flags
